src/Language/PureScript/Parser/Declarations.hs

```diff
@@ -182,8 +182,8 @@ parseTypeClassDeclaration = do
     return implies
   className <- indented *> properName
   idents <- P.many (indented *> kindedIdent)
-  members <- P.option [] . P.try $ do
-    indented *> reserved "where"
+  members <- P.option [] $ do
+    P.try $ indented *> reserved "where"
```

Comment:
Do we need this `try`? `where` should either consume everything or nothing, there should be no reason to backtrack.